src/lighting/ChromaGradientController.cpp

```diff
@@ -85,18 +85,24 @@ Sombrero::FastColor ChromaGradientController::AddGradient(ChromaEventData::Gradi
 }
 
 void Chroma::ChromaGradientController::Update() {
+    getLogger().debug("A");
+    return;
```

Comment:
Remove debug logs and `return`

Reply:
I forgot I added this to temporarily fix a crash 🤦